agent_build_refactored/managed_packages/steps/download_build_dependencies/download_build_dependencies.sh

```diff
@@ -114,72 +161,63 @@ Huqba4uzzTOVO3KXwQEwEJhQVY1MssUXyjnvxRnfvYt22Y1+KOuv2Ps+DCw8J0Gs
 pJIb4TMNGOA95S/JNQzSzHrHir2EID0KSKTzlLjdQNMa1/It1vgh6IEPidai8ik/
 sMB+Qkul/Hc+0BLBe+OVT6Ml1IAL
 =AXHk
------END PGP PUBLIC KEY BLOCK-----"
+-----END PGP PUBLIC KEY BLOCK-----
+" | gpg --import
 
-gpg --import <<< ${GNU_PUBLIC_KEY}
 
 GNU_KEYRING_PATH="${SOURCE_ROOT}/agent_build_refactored/managed_packages/steps/download_build_dependencies/gnu-keyring.gpg"
 
-#mkdir "${STEP_OUTPUT_PATH}/texinfo"
-#pushd "${STEP_OUTPUT_PATH}/texinfo"
-#TEXTINFO_VERSION="6.8"
-#curl -L "https://ftp.gnu.org/gnu/texinfo/texinfo-${TEXTINFO_VERSION}.tar.gz" > texinfo.tar.gz
-#curl -L "https://ftp.gnu.org/gnu/texinfo/texinfo-${TEXTINFO_VERSION}.tar.gz.sig" > texinfo.tar.gz.sig
-#
-#gpg --verify --keyring "${SOURCE_ROOT}/agent_build_refactored/managed_packages/steps/download_build_dependencies/gnu-keyring.gpg" \
-#  texinfo.tar.gz.sig \
-#  texinfo.tar.gz
+mkdir "${STEP_OUTPUT_PATH}/texinfo"
+pushd "${STEP_OUTPUT_PATH}/texinfo"
+curl -L "https://ftp.gnu.org/gnu/texinfo/texinfo-${TEXINFO_VERSION}.tar.gz" > texinfo.tar.gz
+curl -L "https://ftp.gnu.org/gnu/texinfo/texinfo-${TEXINFO_VERSION}.tar.gz.sig" > texinfo.tar.gz.sig
```

Comment:
To be on the safer side, we should likely bundle signatures either directly in the script or in files somewhere in the repo.

Since if we download signature file from the same location from where we download the software artifact, it doesn't really add any additional security since that file could have been compromises / similar as well :)